lib/lrama/state.rb

```diff
@@ -23,6 +23,12 @@ def initialize(id, accessing_symbol, kernels)
       @conflicts = []
       @resolved_conflicts = []
       @default_reduction_rule = nil
+      @predecessors = []
+      @lalr_isocore = self
```

Comment:
[note]: 

* `@lalr_isocore` is an implementation of `lalr1_isocores` (Definition 3.36)
* `@ielr_isocores` is an implementation of `isocore_nexts` (Definition 3.37)
* Their initialization logic is written in Definition 3.45 (split_states).